static/js/components/tiles/histogram_tile.tsx

```diff
@@ -68,12 +69,12 @@ export function HistogramTile(props: HistogramTilePropType): JSX.Element {
   }, [props, rawData]);
 
   useEffect(() => {
-    if (histogramData) {
+    if (histogramData && !_.isEmpty(histogramData.dataPoints)) {
```

Comment:
nit: maybe make a function shouldShowHistogram() & use it both here and in line 77

Reply:
done!